composer.json

```diff
@@ -75,9 +75,11 @@
         "test:cs": "phpcs",
         "test:types": "phpstan analyse --ansi --memory-limit 256M",
         "test:unit": "phpunit --colors=always -d memory_limit=1408M",
+        "test:application": "tests/laravel-test.sh",
         "test": [
             "@test:types",
-            "@test:unit"
+            "@test:unit",
+            "@test:application"
```

Comment:
Would you reconsider merging this pull request if I remove this line? That way, the end-to-end tests are not run by default but are possible to run if needed.